geth/account/accounts.go

```diff
@@ -48,14 +48,14 @@ func NewManager(geth GethServiceProvider) *Manager {
 // sub-account derivations)
 func (m *Manager) CreateAccount(password string) (address, pubKey, mnemonic string, err error) {
 	// generate mnemonic phrase
-	mn := extkeys.NewMnemonic(extkeys.Salt)
+	mn := extkeys.NewMnemonic()
 	mnemonic, err = mn.MnemonicPhrase(extkeys.EntropyStrength128, extkeys.EnglishLanguage)
 	if err != nil {
 		return "", "", "", fmt.Errorf("can not create mnemonic seed: %v", err)
 	}
 
 	// generate extended master key (see BIP32)
-	extKey, err := extkeys.NewMaster(mn.MnemonicSeed(mnemonic, password), []byte(extkeys.Salt))
+	extKey, err := extkeys.NewMaster(mn.MnemonicSeed(mnemonic, ""))
```

Comment:
Why is the empty password used here?

Reply:
@pilu I wonder if this should be configurable. Some people may want to opt for an even more secure way of generating their mnemonic seeds.

Reply:
@adambabik yes, that's my idea of creating another swarm about this. And adding an "advanced option", so that an expert user can add an extra password (after the one used to encrypt the private key) as a salt for the seed generation.
I think this swarm and feature can be added in the future because it's not going to have breaking changes. So we could have more time to discuss about UX/UI.
With this current PR, we will instead have breaking changes, so I think the sooner we merge it, the better, especially if we want to go for mainnet. what do you think @adambabik ?

Reply:
I totally agree!

Reply:
Btw. can we add a comment why we use an empty password instead of `password`? Or in other words, why is the `password` argument ignored.

Reply:
@adambabik  Yes I'm going to do it!